ddtrace/internal/remoteconfig/client.py

```diff
@@ -357,7 +358,8 @@ def _process_response(self, data):
         if last_targets_version is None or targets is None:
             log.debug("No targets in configuration payload")
             for callback in self._products.values():
-                callback(None, None)
+                if callback:
```

Comment:
do we want a try/except here? should we call every callback for all products?

I see below if we call with `callback(config, callback_action)` we have a `try/except` with a `continue`. Do we need the same here?

Reply:
Probably yes (need to ask @avara1986 ) wil preemptively add it.